### ISSUE TITLE
ESP32 Device Id for Traccar

### ISSUE THREAD

**Issue by VaibhavPatelSW** [onlinegill/ESP32-Traccar-GPS#1]
Hi. I am working on the code and unable to figure out my ESP32 device id. Is it a MAC address of ESP32? Could you please guide me to find the ESP32 Unique device Id  for Traccar server?

**Comment by onlinegill** [comment]
> Hi. I am working on the code and unable to figure out my ESP32 device id. Is it a MAC address of ESP32? Could you please guide me to find the ESP32 Unique device Id for Traccar server?

It's id of your traccar server device. When you add a device on traccar server.

**Comment by VaibhavPatelSW** [comment]
When we add device on Traccar Server, it asks Name and Identifier. Did you mean that I can use any custom Identifier to add the device in Traccar Server, and use same Identifier as a Device Id in the code you shared?

**Comment by onlinegill** [comment]
yes, Identifier is device id. Also I would advise to use  ttgo sim7000G so you can put sim card.

https://github.com/onlinegill/LILYGO-TTGO-T-SIM7000G-ESP32-Traccar-GPS-tracker

**Comment by VaibhavPatelSW** [comment]
OK, I'll try ttgo sim7000G. I tried same identifier as Device id in code, still unable to receive gps data to server. Device is showing offline.I am getting message 'DATA SENT TO THE SERVER' but  There is no update in Traccar Server log.

I am using Traccar server on Ubuntu VPS at port 8099, and Neom8n GPS receiver with ESP32

**Comment by onlinegill** [comment]
You need to open port 8099. For testing install traccar app on phone and see if your phone connects.
